launchlib/launcher.go

```diff
@@ -265,3 +281,18 @@ func createReplacer() *strings.Replacer {
 func delim(str string) string {
 	return fmt.Sprintf("%s%s%s", TemplateDelimsOpen, str, TemplateDelimsClose)
 }
+
+func filterHeapArgs(args []string) []string {
+	var filtered []string
+	for _, arg := range args {
+		if !isHeapArg(arg) {
+			filtered = append(filtered, arg)
+		}
+	}
+	return filtered
+}
+
+func isHeapArg(arg string) bool {
+	var lowerCase = strings.ToLower(arg)
+	return strings.HasPrefix(lowerCase, "-xmx") || strings.HasPrefix(lowerCase, "-xms")
```

Comment:
I think case is required to match `-Xmx` and `-Xms`

Reply:
Oh you mean nobody would do -xmx? fair, I wasn't sure if this stuff is case sensitive